SU-SWS/jumpstart_ui
# D8CORE-2729 Allow hero banner overlay to positioned on the right

# READY FOR REVIEW

# Summary
- Add paragraph behavior to expose the option to position the overlay text on the right for hero banners.

# Need Review By (Date)
- 5/5

# Urgency
- medium

# Steps to Test
1. checkout https://github.com/SU-SWS/stanford_profile/pull/407
2. checkout this branch
3. `drush cr; drush cim -y`
4. create a basic page and add a banner to the top or the main components
5. edit the "Style" or "Behaviors" (depending where you added the banner) and change the overlay position to "Right"
6. save and verify the overlay text displays on the right of the image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
